### PR TITLE
Make develop point to kuberentes namespace.

### PR DESCRIPTION
**Pull request by mbruzek**
@chuckbutler can you give this a look and tell me if there are any additional changes you would like to make?

### CODE REVIEW

File: specs/develop.yaml [hunk 1]
```diff
@@ -4,29 +4,29 @@ kubernetes:
...
       num_units: 2
       to: [flannel, flannel]
       annotations:
-        "gui-x": "1219.8695068359375"
-        "gui-y": "430.04345703125"
+        "gui-x": "300"
+        "gui-y": "300"
     etcd:
-      charm: cs:~hazmat/trusty/etcd
+      charm: cs:~kuberentes/trusty/etcd
```
Comment (lazypower) [inline specs/develop.yaml]: Should read 'kubernetes/trusty'

**Comment by lazypower** [comment]
Fix the naming of the etcd charm and i'm +1, I specifically left the develop.yaml as a hands off bundle since I knew you were working with it.

You can see the upcoming changes (minus the proper kubernetes charm location - pending a merge) here https://github.com/whitmo/bundle-kubernetes/pull/40/files

**Comment by mbruzek** [comment]
@chuckbutler thank you for the review.

I plan to move the develop.yaml bundle to what ever the supported bundle is in the future.  For now I just don't want to keep pointing to an old unsupported branch on those 2 charms.

**Comment by lazypower** [comment]
:+1:

**Pull request merged**